dbschema/project.esdl

```diff
@@ -60,6 +60,13 @@ module default {
         message := "A Project's own context should be itself (no more or less)"
       )
     );
+
+    trigger createBudgetOnInsert after insert for each do (
+      insert default::Budget {
+        project := __new__,
```

Comment:
Triggers appear to have a hard time with default expressions.
I had to add it here too
https://github.com/SeedCompany/cord-api-v3/blob/9f47163ab06438265a7e8dff549522ab74bcc596/dbschema/engagement.esdl#L130-L130